modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/TxManagerImpl.java

```diff
@@ -192,6 +196,11 @@ public class TxManagerImpl implements TxManager, NetworkMessageHandler {
 
     private final ReplicaService replicaService;
 
+    /** Registry of locally started active transactions. */
+    private final Map<UUID, InternalTransaction> transactions = new ConcurrentHashMap<>();
```

Comment:
Seems we already have tx states storage VolatileTxStateMetaStorage

Reply:
It need to be filtered by TxStateMeta#txCoordinatorId

Reply:
Please check my [answer below](https://github.com/apache/ignite-3/pull/4690#discussion_r1836119021)

Reply:
The PR was reworked so that the system view now uses existing collections.